<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ders_islemleri</class>
 <widget class="QWidget" name="ders_islemleri">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1047</width>
    <height>786</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_kod">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>130</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_kod">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>140</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ad">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>180</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ad">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>180</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ekle">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>310</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_guncelle">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>310</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sil">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>400</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_tumders">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>481</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_tumogrenci">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>500</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tumders">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_tumogr">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>530</y>
     <width>771</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
